definitions/npm/bitcoinjs-lib_v2.x.x/flow_>=v0.17.x/bitcoinjs-lib_v2.x.x.js

```diff
@@ -97,6 +97,8 @@ declare module 'ecurve' {
 }
 // ---------- copypasta end ---- 
 
+import {Point as ECPoint} from 'ecurve';
```

Comment:
Hm. Seems you are right and it's not possible to do this, since it doesn't typecheck correctly.

So I will have to move `ecurve.Point` outside of `ecurve` and refactor it out into `$npm$ecurve$Point` or something similar. Well, I am not looking forward to that. :)